apps/condo/domains/acquiring/schema/AcquiringIntegration.js

```diff
@@ -73,19 +73,11 @@ const AcquiringIntegration = new GQLListSchema('AcquiringIntegration', {
             isRequired: true,
         },
 
-        supportedBillingIntegrations: {
-            schemaDoc: 'List of supported billing integrations. If one of them is here, it means that this acquiring can accept receipts from it',
-            type: Relationship,
-            ref: 'BillingIntegration',
+        supportedBillingIntegrationsGroup: {
+            schemaDoc: 'Supported billing integrations group. Useful when you need to restrict this acquiring to accept payment only from certain billing',
+            type: Text,
```

Comment:
Type select and options here

Reply:
No, since options are dynamic and set by the user -- it just needs to be a simple string

Reply:
You've just hard-coded this in consts....What should be set by user?

Reply:
But only the default value is hardcoded in const. This was made in order to make this field non-nullable 

**My point is:**
- This mechanism should be simple: you should be able to restrict certain billing from certain acquiring from admin panel anytime w/o making changes to repository (and need to wait to release or hotfix) and preferably w/o changes to `.env` or `database`
- This mechanism should include failsafes (example: Support accidently changes supportedIntegrationsGroup or billingIntegrationGroup)  
- This mechanism needs to be well-documented

Reply:
I can think of some solutions that would satisfy all these requirements: 

1. Text field with a lot validations: Group names are only lowercase latin chars, and `acquiring.supportedGroup` should exist 
2. Select with options from `.env` and validations: `acquiring.supportedGroup` shoud exist
3. A new model called `BillingIntegrationGroup` 

I prefer 1 to 2 (since I don't like changing `.env`), and think 3 would be bad solution. Wyt?